bots/example_js/nav.js

```diff
@@ -83,11 +89,11 @@ nav.getDir = (start, target) => {
 };
 
 nav.isPassable = (loc, fullMap, robotMap) => {
-    const {x, y} = loc;
+    const { x, y } = loc;
     const mapLen = fullMap.length;
-    if (x >= mapLen || x < 0) {
+    if (x >= mapLen - 1 || x < 0) { // BUG HERE, if x = mapLen, then it's off the map!
```

Comment:
I'm pretty sure we don't want to do -1 here. If x = 32 it would be off map which should return false, but 31 should return true. However, here 31 >= 32 - 1 is true, so isPassable would return false.
```suggestion
    if (x >= mapLen || x < 0) {
```